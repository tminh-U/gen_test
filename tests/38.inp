13
9 4 2 11 16 11 3 10 11 20 14 12 8 
5
13 13
4 9
3 10
12 12
11 13
