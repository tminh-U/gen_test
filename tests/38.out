2
16
64
2
4
